### PR TITLE
Feature/move render expression to Dhall.Utility

### PR DESCRIPTION
**Pull request by MarcosJLR**
Created a function to render expression to avoid code repetition as mentioned in issue #2233

### CODE REVIEW

File: dhall-json/json-to-dhall/Main.hs [hunk 1]
```diff
@@ -11,22 +11,17 @@ import Control.Exception   (SomeException, throwIO)
 import Data.Text           (Text)
 import Data.Version        (showVersion)
 import Dhall.JSONToDhall
+import Dhall.Util          (renderExpression)
```
Comment (sjakobi) [inline dhall-json/json-to-dhall/Main.hs:14]: I think we've been using import lists mostly for types and common functions, mostly from `base`.

For an "uncommon" utility like this one I'd suggest rather using a qualified import.

File: dhall-yaml/yaml-to-dhall/Main.hs [hunk 2]
```diff
@@ -12,21 +12,16 @@ import Data.Text           (Text)
 import Data.Version        (showVersion)
 import Dhall.JSONToDhall   (Conversion, parseConversion)
 import Dhall.Pretty        (CharacterSet (..))
+import Dhall.Util          (renderExpression)
```
Comment (sjakobi) [inline dhall-yaml/yaml-to-dhall/Main.hs:15]: (Dito)

**Review by sjakobi — COMMENTED** [review]
LGTM apart from a small style issue. :)

File: dhall-json/json-to-dhall/Main.hs (hunk 1 above)
Comment (MarcosJLR) [reply dhall-json/json-to-dhall/Main.hs:14]: Thanks! I didn't know, I'll fix it right away

**Review by sjakobi — APPROVED** [review]

**Comment by MarcosJLR** [comment]
I noticed that some tests have been failing since PR #2196 was merged into master, let me know if I should do something to fix it in this branch or if I should wait for it to be solved in master and then bring those changes to this branch

**Comment by sjakobi** [comment]
@MarcosJLR I believe this was fixed in https://github.com/dhall-lang/dhall-haskell/pull/2232. Maybe try rebasing on the latest `master`.

**Comment by sjakobi** [comment]
If that's not it, please open an issue.

**Pull request merged**